libs/vkd3d/memory.c

```diff
@@ -173,14 +169,10 @@ void vkd3d_free_device_memory(struct d3d12_device *device, const struct vkd3d_de
 }
 
 static HRESULT vkd3d_try_allocate_device_memory(struct d3d12_device *device,
-        VkDeviceSize size, VkMemoryPropertyFlags type_flags, uint32_t base_type_mask,
-        void *pNext, struct vkd3d_device_memory_allocation *allocation)
+        VkDeviceSize size, VkMemoryPropertyFlags type_flags, const uint32_t base_type_mask,
+        void *pNext, bool budgeted, struct vkd3d_device_memory_allocation *allocation)
```

Comment:
Not entirely sure but `budgeted` feels like the wrong word (IIUC it means something that was drafted into the budget). Maybe `use_budget`?